helx/charts/search/values.yaml

```diff
@@ -328,7 +331,7 @@ tranql:
           - Content-Type
       timeout_ms: 0
   existingRedis:
-    host: helx-redis-master
+    host: helx-redis-slave
```

Comment:
Do the redis slaves support write access? or will it only be used for reads?

Reply:
tranql does only reads.